src/main/java/org/galatea/starter/service/IexService.java

```diff
@@ -45,5 +48,47 @@
     }
   }
 
+  /**
+   * Get the historical price data for a symbol over a range of dates
+   *
+   * @param symbol the symbol to get the price data for.
+   * @param range the range option.  See "https://iexcloud.io/docs/api/#historical-prices" for
+   *     options.
+   * @param date the date to get the data from.  YYYYMMDD format.
+   * @return a list of historical price objects for the symbol passed in.
+   */
+  public List<IexHistoricalPrice> getHistoricalPrice(
+      final String symbol,
+      final String range,
+      final String date) {
+    log.info("Retrieving historical price data for symbol: " + symbol + ", range: " + range + ", date: " + date);
```

Comment:
This log library actually supports an easier way to include your own parameters in the log. You can use `{}` in the log string and then just provide the string you want to substitute as an additional paramater.
`log.info("Retrieving historical price data for symbol: {}, range: {}, date: {}", symbol, range, date);`